internal/cmd/init.go

```diff
@@ -11,31 +11,39 @@ import (
 	"github.com/freepik-company/gommitizen/internal/git"
 )
 
+const (
+	cmdInitBumpChangelog = "bump-changelog"
```

Comment:
According to the official style and decisions documentacion for GO lang, `constants`  should not base their names on their own values but in their role. So I propose change this to something like `bumpChangelogFlagName` or `flagBumpChangelog` 

More info:
- https://google.github.io/styleguide/go/decisions#constant-names